fe/fe-core/src/test/java/com/starrocks/sql/plan/ArrayTypeTest.java

```diff
@@ -59,11 +59,11 @@ public static void beforeClass() throws Exception {
     public void testConcatArray() throws Exception {
         String sql = "select concat(c1, c2) from test_array";
         String plan = getFragmentPlan(sql);
-        assertContains(plan, "array_concat(2: c1, CAST(3: c2 AS ARRAY<VARCHAR>))");
+        assertContains(plan, "array_concat(2: c1, CAST(3: c2 AS ARRAY<VARCHAR(65533)>))");
```

Comment:
Why the new type has a length limit ?

Reply:
I think because the parameters(`c1/c2`) of function has length..the type derive will use them